futures-cpupool/src/lib.rs

```diff
@@ -188,12 +202,14 @@ impl CpuPool {
 }
 
 fn work(inner: &Inner) {
+    inner.after_start.as_ref().map(|fun| fun());
```

Comment:
I think "fun" should be renamed to "func".
It's only 1 more character, but much more clear.